boost.yaml

```diff
@@ -59,13 +59,7 @@ pipeline:
   - uses: fetch
     with:
       uri: https://boostorg.jfrog.io/artifactory/main/release/${{package.version}}/source/boost_${{vars.mangled-package-version}}.tar.gz
-      expected-sha256: 2575e74ffc3ef1cd0babac2c1ee8bdb5782a0ee672b1912da40e5b4b591ca01f
-
-  # Apply patches to fix build https://github.com/boostorg/bcp/pull/18
-  - runs: |
-      cd ./tools/bcp
-      patch -p1 < ../../c98516b5b76e9132eba78a399af9c95ec8d23bd4.patch
-      patch -p1 < ../../cd21e9b4a749a77c24facf2da44f01e032c40842.patch
```

Comment:
We should also delete this patch files from boost/ die

Reply:
Yea good call-out, I thought it was a cheery-pick at first, didn't read closely enough into it - will take a look, thanks!